libraries/Microsoft.Bot.Schema/Teams/InvokeResponseBase.cs

```diff
@@ -0,0 +1,43 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Newtonsoft.Json;
+
+namespace Microsoft.Bot.Schema.Teams
+{
+    /// <summary>
+    /// Specifies Invoke response base including response type.
+    /// </summary>
+    public partial class InvokeResponseBase
```

Comment:
Is there a reason to name it as InvokeResponseBase, will we have a class of InvokeResponse? I saw we have both classes for task.

Reply:
This will align with our class design in the reference repo.

Reply:
I can sync up with you offline for this.